### DISCUSSION TITLE
As beginner it's impossible to use this crate

### DISCUSSION THREAD

**Discussion by PhYdrogen** [hunger/downloader#51] [Ideas]
I started to learn rust and i wanted to use this crate but the readme is empty and don't help much... I would like if possible an easy exemple to how this repo/crate works, thank you.

**Comment by hunger** [comment]
Yes, I wrote this for use in one of my projects and basically pushed it to crates.io to make my live easier:-)

It needs work to make that useful for other people, I agree. Contributions are welcome :-)
